website/src/tutorials/tab-feat-attr.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Tabular Feature Attribution
+layout: layouts/tutorial.liquid
+
+hero-image: /assets/images/sample-banner.png
+hero-title: "Tabular Feature Attribution"
+hero-copy: "Learn how to use the Tabular Feature Attribution module in LIT."
+
+bc-anchor-category: "analysis"
+bc-category-title: "Analysis"
+bc-title: "Tabular Feature Attribution"
+
+time: "15 minutes"
+takeaways: "Learn how to use the Kernel SHAP based Tabular Feature Attribution module in LIT."
+---
+
+## Tabular Feature Attribution
+
+{% include partials/link-out link: "../../demos/penguins.html", text: "Explore this demo yourself." %}
+
+Or, run your own with [`examples/penguin_demo.py`](https://github.com/PAIR-code/lit/blob/main/lit_nlp/examples/penguin_demo.py)
+
+LIT supports many techniques like salience maps and counterfactual generators for text data. But what if you have a tabular dataset? You might want to find out which data features are most relevant in the model’s predictions. LIT provides a Feature Attribution module, which is currently supported for [tabular datasets](https://github.com/PAIR-code/lit/wiki/components.md#tabular-data) only, to enable important feature identification. This tutorial provides a walkthrough for this module within LIT, on the [Palmer Penguins dataset](https://allisonhorst.github.io/palmerpenguins/).
```

Comment:
"identification of important features" reads better than "import feature identification"